### DISCUSSION TITLE
Macros message

### DISCUSSION THREAD

**Discussion by purrutia** [folke/noice.nvim#675] [Q&A]
Hi all,

Without the plugin installed, when I start to record a Macro, a message appears at the bottom (like **recording @q**).
Can noice be configured to show this message? Because now when I start a macro I don't have a clue if I press q once or twice or if recording starts or stops.

Thanks!

Solved!
With the lualine example on Status Line section

**Comment by juliogc** [comment]
@purrutia Sorry to bring this thread back from the grave, but could you please share where you found the example that solved the issue? I can’t seem to find it anywhere 🫠

**Comment by snoopdouglas** [comment]
For anyone arriving here from Google, add a route:

```lua
			routes = {
				-- Show macro recording
				{
					view = "notify",
					filter = { event = "msg_showmode" },
				},
			},
```

**Reply by juliogc** [reply]
Yeah, I forgot to update here, but there a recipe with this solution in the Wiki:
https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#show-recording-messages
